### DISCUSSION TITLE
Bug in Query editor? Timestamp with time zone always filled in....

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#1093] [Questions]
Hi,

I created the following table:

```
CREATE TABLE "widgets" (
	"id" bigserial NOT NULL PRIMARY KEY,
	"created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
	"updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
	"deleted_at" TIMESTAMP WITH TIME ZONE,
);

```

and when adding records via the online interface all records directly get an date assigned to "deleted_at" as well, while I would expect a null value by default. Is this a bug in the online editor, or am I doing something silly wrong?

**Comment by silentworks** [comment]
Yeah, this looks like a bug, can you open an issue on the [Issue Tracker](https://github.com/supabase/supabase/issues) please about this.
